docs/README.md

```diff
@@ -438,7 +438,11 @@ ___
 
 Establishes a connection to the target database.
 
-**NOTE** '*LOCAL' can be provided as the `database` when connecting to a local database, allowing `user` & `password` to be optionally passed.
+**NOTE** 
+
+- `*LOCAL` can be provided as the `database` when connecting to a local database, allowing `user` & `password` to be optionally passed.
+
+- To access functions and stored procedures without fully qualifying ensure to set your path via sql statement. `SET PATH MYSCHEMA`

```

Comment:
I'd like to see a link to the actual SQL docs here. https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_74/db2/rbafzsetpath.htm

Reply:
eg. 
To access functions and stored procedures without fully qualifying, ensure that the CURRENT PATH is set correctly using the `SET PATH` SQL statement. See [here](https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_74/db2/rbafzsetpath.htm) for more info.

Reply:
> To access SQL functions and stored procedures without fully qualifying them, ensure that the CURRENT PATH is set correctly using the SET PATH SQL statement. See here for more info.

Well said